src/cfml/system/util/TablePrinter.cfc

```diff
@@ -339,28 +349,20 @@ component {
 			return arguments.text;
 		} else if( textLength > arguments.maxWidth ) {
 			if( arguments.maxWidth < 4 ) {
-				return left( text, arguments.maxWidth );	
+				return left( text, arguments.maxWidth );
 			} else {
-				return left( text, arguments.maxWidth-3 )&'...';				
+				return left( text, arguments.maxWidth-3 )&'...';
 			}
 		}
 		arguments.text &= repeatString( arguments.padChar, arguments.maxWidth-textLength );
 		return arguments.text;
 	}
-	
+
 	function stringify( any data ) {
 		if( isSimpleValue( data ) ) {
 			return data;
-		} else if ( isArray( data ) ) {
-			return '[Array]';
-		} else if ( isStruct( data ) ) {
-			return '[Struct]';
-		} else if ( isXML( data ) ) {
-			return '[XML]';
-		} else if ( isBinary( data ) ) {
-			return '[Binary]';
 		} else {
-			return '[#data.getClass().getName()#]';			
+			return SerializeJSON(data);
```

Comment:
@scottsteinbeck I'm not sure about this.  This else could be stuff like raw java objects, etc or something that doesn't serialize well.  Is there a particular sort of data you want to serialize, and can we check for it explicitly and leave the default 'else' as-is?